lib/routes/dev/developers_view.dart

```diff
@@ -109,4 +114,26 @@ class DevelopersView extends StatelessWidget {
     final zipFile = XFile(zipFilePath);
     Share.shareXFiles([zipFile]);
   }
+
+  void _exportStaticBackup(BuildContext context) async {
+    final accBloc = context.read<AccountBloc>();
+    const name = "emergency.recover";
+    final staticBackup = await accBloc.exportStaticChannelBackup();
+
+    if (staticBackup.backup != null) {
+      final backup = staticBackup.backup;
+
+      final emergencyList = backup!.expand((hex) => HEX.decode(hex)).toList();
+
+      Directory workingDir = await getApplicationDocumentsDirectory();
+      String filePath = '${workingDir.path}/$name';
+      File file = File(filePath);
+      await file.writeAsBytes(emergencyList, flush: true);
+      final storageFile = XFile(filePath);
+      Share.shareXFiles([storageFile]);
+    } else {
+      // ignore: use_build_context_synchronously
+      showFlushbar(context, title: "No active channels");
```

Comment:
Maybe "`Static backup data is missing`" ?

Reply:
Added